app/models/answer.rb

```diff
@@ -1,5 +1,7 @@
 class Answer < ActiveRecord::Base
   include VotesCounter
+  include ReputationConcern
```

Comment:
Just thinking that this concern might be better off in the `user` model given that all its operation/effects are reflect on a user object :thought_balloon: